drupal/config/sync/user.role.translator.yml

```diff
@@ -0,0 +1,33 @@
+uuid: 9b47ad8f-6d2b-45ab-8757-116590230bf8
+langcode: cs
+status: true
+dependencies: {  }
+id: translator
+label: Překladatel
+weight: -8
+is_admin: null
+permissions:
+  - 'access administration pages'
+  - 'access site in maintenance mode'
+  - 'access toolbar'
+  - 'change own username'
+  - 'create content translations'
+  - 'translate any entity'
+  - 'translate area taxonomy_term'
+  - 'translate block_content'
+  - 'translate measure node'
+  - 'translate measure_type taxonomy_term'
+  - 'translate notice node'
+  - 'translate page node'
+  - 'translate region taxonomy_term'
+  - 'translate situation node'
+  - 'update content translations'
+  - 'use content transition create_new_draft'
```

Comment:
staci im vediet vytvorit draft? publikovat to bude editor?

Reply:
JJ tak si to představuju a Petr s tím byl v pohodě.

Reply:
Já jsem s tím byl v pohodě, otázka jestli s tím budou v pohodě oni? Ale kdyžtak doplníme.

Reply:
Myslel jsem druhého Petra :D